package.json

```diff
@@ -35,7 +35,7 @@
     "web-vitals": "^2.1.4"
   },
   "scripts": {
-    "start": "rsbuild dev --open",
+    "start": "rsbuild dev",
     "build": "rsbuild build",
     "test": "jest",
```

Comment:
I removed the —open argument because it is another command on the docker linux image and so docker struggled with that conflicting arg to run the app. Future package.json may add a start:dev script that opens automatically